doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,249 @@
 # Rust SDK
 
-More docs coming soon!
+The code for the Rust SDK lives in `/econia/src/rust/sdk`.
+This code provides programmatic access to Econia exchanges, in addition to offering an example to put it all together shown later in this document.
+
+# Structure
+
+- The main struct of the SDK is `EconiaClient`.
+  This allows you to submit transactions, query events and get a view client (`EconiaViewClient`).
+  To create a new `EconiaClient`, you need a node URL, the Econia contract's address, an `aptos_sdk::types::LocalAccount` and you can optionally pass a client config.
```

Comment:
For consistency with the Move docs, how about?

"the Econia package address"

Also a comma is missing after `LocalAccount`